src/selectedtests/task_mappings/get_task_mappings.py

```diff
@@ -2,21 +2,26 @@
 from decimal import Decimal
 from typing import List
 
-from pymongo.collection import Collection
+import inject
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.injection_helper import remap_for_injection
 
 
+@inject.autoparams()
+@remap_for_injection
 def get_correlated_task_mappings(
-    collection: Collection, changed_source_files: List[str], project: str, threshold: Decimal
+        db: MongoWrapper, changed_source_files: List[str], project: str, threshold: Decimal
 ) -> List[dict]:
     """
     Retrieve task mappings associated with a given evergreen project and list of source files.
 
     :param project: The name of the evergreen project to analyze.
-    :param collection: Collection to act on.
+    :param db: db containing task mappings collection.
     :param changed_source_files: List of source files for which task mappings should be retrieved.
     :param threshold: Min threshold desired for flip_count/source_file_seen_count ratio.
     :return: A list of task mappings for the changed files.
     """
+    collection = db.task_mappings()
```

Comment:
This makes things easier. Arguably, searching for correlated task mappings in a collection other than `db.task_mappings()` doesn't really make sense anyway.

Reply:
Similar logic below for inserting things into/getting things from collections.